### PR TITLE
Create Security and Response policy

### PR DESCRIPTION
**Pull request by FerralCoder**

### CODE REVIEW

**Review by andreslagarcavilla — APPROVED** [review]

**Comment by FerralCoder** [comment]
This PR goes along with https://github.com/chipsalliance/Caliptra/pull/83

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Caliptra Project Security Incident Response
+
+Please refer to the security policy at [Caliptra security policy](https://github.com/chipsalliance/caliptra/security/policy).
```
Comment (andreslagarcavilla) [inline SECURITY.md:3]: Actually: isn't this Caliptra/security.md

**Review by andreslagarcavilla — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (FerralCoder) [reply SECURITY.md:3]: The link above is correct. The page will contain the security policy and other useful links once SECURITY.md is merged into the main Caliptra repo.

Comment (FerralCoder) [reply SECURITY.md:3]: Here's what the equivalent page looks like in my sample repo: https://github.com/FerralCoder/Test/security/policy

**Review by andreslagarcavilla — APPROVED** [review]

**Review by JohnTraverAmd — APPROVED** [review]

**Pull request merged**